### PR TITLE
[6.5.x] RHBPMS-4224, JBPM-5303 - Field 'contentMap' from a task content acces…

### PR DESCRIPTION
**Pull request by mswiderski**
…sed using REST is not being filled(or coming null)

### CODE REVIEW

**Comment by mswiderski** [comment]
test failures certainly not related to the PR

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
@psiroky could you please take a quick look?

File: kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/RestTaskAndAuditDeploymentIdTest.java [hunk 1]
```diff
@@ -125,5 +144,46 @@ public void testRestUrlProcessTaskProcessing() throws Exception {
...
+               @SuppressWarnings("rawtypes")
+            ExecuteAndSerializeCommand command = (ExecuteAndSerializeCommand) invocation.getArguments()[1];
+               
+               return (JaxbContent) command.execute(null);
+            }
+        }).when(userTaskServiceMock).execute(any(String.class), any(ExecuteAndSerializeCommand.class));
+
+        TaskCommand<Content> cmd = Mockito.mock(GetContentByIdCommand.class);
+        Content content = Mockito.mock(Content.class);
+        when(content.getId()).thenReturn(1l);
```
Comment (psiroky) [inline kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/RestTaskAndAuditDeploymentIdTest.java]: Nit picking, but `1l` looks awfully like `11`. I am usually using`1L`.

**Comment by psiroky** [comment]
The code looks good, but I can't really confirm if this is the only place that needs changes (e.g. if some other marshalling code needs adjustment as well -- I simply don't have knowledge about the whole thing).

**Pull request merged**